### ISSUE TITLE
[SELECT] Instantly closes after opening when using touch based screen readers

### ISSUE THREAD

**Issue by floppey** [radix-ui/primitives#1343]
### Current Behavior

When we're testing the component using a iphone/ipad with voiceover we are unable to select an option. The select closes itself as soon as it has opened.

### Expected behavior

The select should remain open when using touch based screen readers

### Reproducible example

This behaviour can be reproduced in the example on radix-ui.com
https://www.radix-ui.com/docs/primitives/components/select

### Suggested solution

I have not had a chance to dig down into the code to find a solution

### Your environment

| Software         | Name(s) | Version |
| ---------------- | ------- | ------- |
| Radix Package(s) |    @radix-ui/react-select     |   0.1.1      |
| React            | n/a     |   17.0.2      |
| Browser          |    IOS Safari     |    latest     |
| Assistive tech   |    Voiceover     |         |
| Node             | n/a     |     14.18.1    |
| npm/yarn         |   yarn      |   1.22.15      |
| Operating System |    Windows     |   10 enterprise       |

**Comment by innub** [comment]
Hey @floppey, 

I'm not sure if this was issue was already resolved, but I was unable to reproduce this issue on Safari 14, an older version than the one you specified. It might be an issue unique to Safari 15. I have no access to higher versions of Safari, but I hope someone else is able to resolve this!

:pray:

**Comment by benoitgrelard** [comment]
We haven't had a chance to look at this yet, but thanks for the extra findings @innub.

**Comment by Watso196** [comment]
Are there any updates to this issue? My team is also running into these issues. 

We found that setting `onPointerDownOutside` on the menu content component to a function that prevents the default functionality, that allowed us to select an option. That allowed users to select an option. We did still struggle with selecting the correct option via VoiceOver on iOS until making the menu options larger. Together those provided accurate access to the options via VoiceOver iOS and Android TalkBack as well.
